### PR TITLE
docs: explain better how to get logs when self-hosting

### PR DESCRIPTION
**Pull request by HonkingGoose**
## Changes

- Drop explanation in bot comment, instead link to the [Renovate troubleshooting docs for self-hosted](https://docs.renovatebot.com/troubleshooting/#self-hosted)
- Explain better how to get the logs when self-hosting

## Context

- https://github.com/renovatebot/renovate/discussions/27146#discussioncomment-8416122
- https://github.com/renovatebot/renovate/discussions/27146#discussioncomment-8417365

I don't self-host or deal with logging much, so this is a draft PR, with the hope that the maintainers suggest changes to finish the PR. :smile: 

## Documentation (please check one with an [x])

- [ ] I have updated the documentation, or
- [x] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: .github/label-actions.yml [hunk 1]
```diff
@@ -64,6 +64,13 @@
 
     If you're running self-hosted, run with `LOG_LEVEL=debug` in your environment variables and search for whatever dependency/branch/PR that is causing the problem.
 
+    Add steps how to sufficiently enable logging.
```
Comment (HonkingGoose) [inline .github/label-actions.yml:67]: Hmm, maybe we should drop the text instructions here, and link to [Renovate docs, troubleshooting, self-hosted section](https://docs.renovatebot.com/troubleshooting/#self-hosted) instead?

That way we have _one place_ for the information, and people will hopefully read the docs and complain if it's not clear. :smile:

Comment (p3k) [reply .github/label-actions.yml:67]: Good idea – however, I still would not know how to exactly create the (requested) plain log file from it.

As I already mentioned in the thread given as context, I was adopting the settings from the [official Renovate runner for GitLab](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/renovate.gitlab-ci.yml?ref_type=heads#L8) – and the resulting log file in ndjson format was frowned upon.

Comment (HonkingGoose) [reply .github/label-actions.yml]: I dropped the explanation in the bot comment, and instead link to the Renovate docs. I've moved the draft text to the troublehooting docs.

File: docs/usage/troubleshooting.md [hunk 2]
```diff
@@ -35,6 +35,13 @@ For troubleshooting it's recommended to increase logging to `DEBUG` level by add
 If your Renovate logs are being processed by a log service before you access them, you may find it better to have Renovate output logs in JSON format instead so that they can be reliably parsed and filtered.
 This can be achieved by adding `LOG_FORMAT=json` to your environment variables before invoking Renovate.
 
+Add steps how to sufficiently enable logging.
+
+Add steps how to save the log file during a CI run.
+
+Add steps how to examine / post ndjson output (filetype is kind of exotic).
+@viceice said we prefer plain JSON?
```
Comment (HonkingGoose) [inline docs/usage/troubleshooting.md:43]: > Good idea – however, I still would not know how to exactly create the (requested) plain log file from it.
> 
> As I already mentioned in the thread given as context, I was adopting the settings from the [official Renovate runner for GitLab](https://gitlab.com/renovate-bot/renovate-runner/-/blob/main/templates/renovate.gitlab-ci.yml?ref_type=heads#L8) – and the resulting log file in ndjson format was frowned upon.

Copy/pasting the comment from @p3k so we don't miss it. 😉

**Comment by rarkins** [comment]
Right now this PR only containers placeholders, and it hasn't progressed in two months, so I think it's best to close it so it doesn't stay open forever. It can be replaced with an issue if desired

**Comment by HonkingGoose** [comment]
> Right now this PR only containers placeholders, and it hasn't progressed in two months, so I think it's best to close it so it doesn't stay open forever. It can be replaced with an issue if desired

The idea was to get help from the maintainers to fill it out. I'll replace it with an issue then. 😄

**Pull request closed without merging**